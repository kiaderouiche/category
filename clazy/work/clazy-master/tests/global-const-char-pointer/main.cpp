



static const char * const foo  = "1";
static const char * foo2 = "2";
const char * foo3 = "3"; // Warning

const char * const foo4 = "4";
static const char foo5[] ="5";
const char foo51[] ="5";

namespace {
    const char * foo6 = "6";
}

void test()
{
    const char * foo6 = "6";
}

extern const char *externFoo;
