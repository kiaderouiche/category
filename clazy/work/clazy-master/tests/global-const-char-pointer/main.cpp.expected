global-const-char-pointer/main.cpp:7:1: warning: non const global char * [-Wclazy-global-const-char-pointer]
